.github/workflows/build-binaries.yml

```diff
@@ -534,8 +540,6 @@ jobs:
           - target: aarch64-unknown-linux-musl
             arch: aarch64
             maturin_docker_options: -e JEMALLOC_SYS_WITH_LG_PAGE=16
-          - target: armv7-unknown-linux-musleabihf
-            arch: armv7
```

Comment:
Should we build this here, or in the `linux-arm` category that I augmented above?